src/super_gradients/training/datasets/segmentation_datasets/segmentation_dataset.py

```diff
@@ -77,26 +50,8 @@ def __init__(self, root: str, list_file: str = None, samples_sub_directory: str
                                       sample_loader=self.sample_loader, sample_transform=self.sample_transform,
                                       target_loader=self.target_loader, target_transform=self.target_transform,
                                       collate_fn=collate_fn)
-        # DEFAULT TRANSFORMS
-        # FIXME - Rescale before RandomRescale is kept for legacy support, consider removing it like most implementation
-        #  papers regimes.
-        default_image_mask_transforms_aug = transform.Compose([RandomFlip(),
-                                                               Rescale(short_size=self.img_size),
-                                                               RandomRescale(scales=(0.5, 2.0)),
-                                                               RandomRotate(),
-                                                               PadShortToCropSize(self.crop_size),
-                                                               CropImageAndMask(crop_size=self.crop_size,
-                                                                                mode="random"),
-                                                               RandomGaussianBlur()])
-
-        self.image_mask_transforms_aug = image_mask_transforms_aug or default_image_mask_transforms_aug
-        # FIXME: CROP SIZE CANNOT BE PASSED WHEN LIST
-        if image_mask_transforms is None:
-            image_mask_transforms = transform.Compose([Rescale(short_size=self.crop_size),
-                                                       CropImageAndMask(crop_size=self.crop_size, mode="center")
-                                                       ])
-
-        self.image_mask_transforms = image_mask_transforms
+
+        self.transforms = transform.Compose(transforms) if transforms else []
```

Comment:
move the if statement inside brackets.

Reply:
🤦 My bad. Must have lost this fix somewhere when I reverted/merged...